packages/react-instantsearch-dom/src/components/QueryRuleCustomData.tsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+import { CustomUserData } from 'react-instantsearch-core';
+import { createClassNames } from '../core/utils';
+
+const cx = createClassNames('QueryRuleCustomData');
+
+type QueryRuleCustomDataRenderProps<TItem> = {
+  items: TItem[];
+};
+
+export type QueryRuleCustomDataProps<TItem> = {
+  items: TItem[];
+  className?: string;
+  children: (options: QueryRuleCustomDataRenderProps<TItem>) => React.ReactNode;
+};
+
+const QueryRuleCustomData: React.SFC<
```

Comment:
we use FC in the library so far

```suggestion
const QueryRuleCustomData: React.FC<
```

Reply:
We use `SFC` for `Hits`: https://github.com/algolia/react-instantsearch/blob/642ba0b207deb8534c2d97bcde2f2b1fedd7e48c/packages/react-instantsearch-dom/src/components/Hits.tsx#L22

Apparently it's deprecated.